### PR TITLE
fixes(release 4.25.0)

### PR DESCRIPTION
**Pull request by SorinC6**
https://emurgo.atlassian.net/browse/YOMO-1168

Demo: https://recordit.co/58wGJKjEDG

### CODE REVIEW

**Review by banklesss — APPROVED** [review]

File: apps/wallet-mobile/src/components/Warning/Warning.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import {Icon} from '../Icon'
+import {Spacer} from '../Spacer'
+
+type Props = {content: string}
+
+export const Warning = ({content}: Props) => {
+  return (
+    <View style={styles.notice}>
+      <Icon.Info size={30} color="#ECBA09" />
```
Comment (banklesss) [inline apps/wallet-mobile/src/components/Warning/Warning.tsx]: dont need the theme stuff here ? 🤔

Comment (banklesss) [reply apps/wallet-mobile/src/components/Warning/Warning.tsx]: not sure if its feasible

Comment (SorinC6) [reply apps/wallet-mobile/src/components/Warning/Warning.tsx]: made the change now, was not making the change as the current colors were not in the theme but have changed them to match the ones we have now

Comment (banklesss) [reply apps/wallet-mobile/src/components/Warning/Warning.tsx]: perfect !

**Pull request merged**